packages/vite/package.json

```diff
@@ -32,7 +32,8 @@
     "@nrwl/devkit": "file:../devkit",
     "@nrwl/workspace": "file:../workspace",
     "@swc/helpers": "^0.4.11",
-    "chalk": "4.1.0"
+    "chalk": "4.1.0",
+    "vite": "3.1.8"
```

Comment:
We don't want to depend on `vite`, instead we should be installing it to the user's workspace.

There should be versions set in `packages/vite/src/utils/version.ts` and we init with those.

Reply:
It should still be a peer dep right?